website/blog/2018-05-07-using-typescript-with-react-native.md

```diff
@@ -10,7 +10,17 @@ category: engineering
 
 JavaScript! We all love it. But some of us also love [types](https://en.wikipedia.org/wiki/Data_type). Luckily, options exist to add stronger types to JavaScript. My favourite is [TypeScript](https://www.typescriptlang.org), but React Native supports [Flow](https://flow.org) out of the box. Which you prefer is a matter of preference, they each have their own approach on how to add the magic of types to JavaScript. Today, we're going to look at how to use TypeScript in React Native apps.
 
-We'll be using Microsoft's [TypeScript-React-Native-Starter](https://github.com/Microsoft/TypeScript-React-Native-Starter) repo as a guide.
+This post uses Microsoft's [TypeScript-React-Native-Starter](https://github.com/Microsoft/TypeScript-React-Native-Starter) repo as a guide.
+
+**Update**: Since this blog post was written, [Babel 7 was released with TypeScript support built-in](https://blogs.msdn.microsoft.com/typescript/2018/08/27/typescript-and-babel-7/). Babel 7 replaces all the set up described in this blog post with just one command:
+
+```sh
+react-native init MyAwesomeProject --template typescript
+```
+
+However, there _are_ some limitations to Babel's TypeScript support, which the blog post above goes into in detail. The steps outlined in _this_ post still work, and Artsy is still using [react-native-typescript-transformer](https://github.com/ds300/react-native-typescript-transformer) in production, but the fastest way to get up and running with React Native and TypeScript is using the above command. You can always switch later if you have to, and you can read [my colleague's blog post on Babel 7 TypeScript workarounds](http://artsy.github.io/blog/2017/11/27/Babel-7-and-TypeScript/) if you get stuck.
```

Comment:
The Microsoft link up above is more accurate (and recent) and should probably replace Artsy's blog post here. That was written when Babel 7 was in the early beta phase and there were lots of bugs and issues to be worked out, with very few setup guides online.